docker/load_mongodump.sh

```diff
@@ -10,6 +10,7 @@ CONFIG_FILE="$SCRIPT_DIR/../docker-compose-dev.yml"
 if [ "$#" -ne 1 ]; then
     echo "Usage: $0 <mongodump-file>"
     echo "  <mongodump-file> : The path to the MongoDB dump file to be restored."
+    echo " run git add -f <docker compose file> after using this command"
```

Comment:
@TeachMeTW the rest of this change is fine, but this seems weird. Why are we putting in instructions for the **uncommon** case into the load script? If people change the `docker-compose.dev.yml` because of the load script, they shouldn't be checking it in.

They should only be checking it in if they went in and edited it directly.

Reply:
@shankari Fixed it, good catch, I think I autopiloted that line